engine.go

```diff
@@ -161,6 +164,20 @@ func (c *EngineCmd) startRPC(ctx context.Context) {
 	if err != nil {
 		c.log.Fatal(err)
 	}
+
+	rpcSrv.RegisterName("eth", c.backend)
+	if err := node.RegisterApis([]rpc.API{
+		{
+			Namespace:     "eth",
+			Version:       "1.0",
+			Service:       c.backend,
```

Comment:
Doesn't this expose the engine RPC methods in the eth namespace by accident this way? If the engine RPC endpoint is authenticated, but the eth RPC endpoint is not, then this is a security issue. (doesn't really matter for this test tool, but sets a bad example).

Reply:
Yeah, I'll move the thing to its own backend, since it's polluting the engine space in addition to exposing the engine.